migrating_from_ocp_3_to_4/planning-migration-3-4.adoc

```diff
@@ -21,7 +21,7 @@ With {product-title} 3, administrators individually deployed {op-system-base-ful
 
 {product-title} 4 represents a significant change in the way that {product-title} clusters are deployed and managed. {product-title} 4 includes new technologies and functionality, such as Operators, machine sets, and {op-system-first}, which are core to the operation of the cluster. This technology shift enables clusters to self-manage some functions previously performed by administrators. This also ensures platform stability and consistency, and simplifies installation and scaling.
 
-Beginning with {product-title} 4.13, {op-system} now uses {op-system-base-full} 9.2 packages. This enhancement enables the latest fixes and features as well as the latest hardware support and driver updates. For more information about how this upgrade to RHEL 9.2 might affect your options configuration and services as well as driver and container support, see the xref:../release_notes/ocp-4-13-release-notes.adoc#ocp-4-13-rhcos-rhel-9-2-packages[RHCOS now uses RHEL 9.2] in the _OpenShift Container Platform 4.13 release notes_.
+Beginning with {product-title} 4.13, {op-system} now uses {op-system-base-full} 9.2 packages. This enhancement enables the latest fixes and features as well as the latest hardware support and driver updates. For more information about how this upgrade to RHEL 9.2 might affect your options configuration and services as well as driver and container support, see the xref:../release_notes/ocp-4-14-release-notes.adoc#ocp-4-14-rhcos-rhel-9-2-packages[RHCOS now uses RHEL 9.2] in the _OpenShift Container Platform 4.13 release notes_.
```

Comment:
Mismatch on the file path and the _OpenShift Container Platform 4.13 release notes_, but should this remain as a link to the 4.13 relnotes specifically, or will this `*-rhcos-rhel-9-2-packages` content maybe be carried over into the 4.14 relnotes?

Reply:
Or another option: maybe this para should just be removed from the 4.14+ docs. 🤷

Reply:
I would just remove it, there is currently no commitment from engineering that RHCOS 4.14 should expect a RHEL bump

Reply:
Perhaps we should keep it with the hard link. If people are going from EUS to EUS versions, they might have missed this change in 4.13.

Reply:
I agree with @jldohmann. We can just remove it for now.

Reply:
For now, I'm going to put the hardcoded link in and we can revisit this closer to GA. I could see a use case for the link in 4.12 to 4.14 updates.